proto/blob/event.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package blob;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/celestiaorg/celestia-app/x/blob/types";
+
+// EventPayForBlob defines an event that is emitted with the signer and blob size
+// at the end of a pay for blob txn
+message EventPayForBlob {
```

Comment:
So we are changing event.type from `payforblob` to `blob.EventPayForBlob` or am I missing something here? If we are changing event type then relevant test (`TestSubmitWirePayForBlob`) in `integration_test.go` can be updated.

Reply:
Yes, there's a pending task in this PR for the same

Reply:
My bad, I overlooked the draft status and checklist (used to GitLab PRs 😅 ). My apologies 🙏